nodes/src/test/java/io/aexp/nodes/graphql/models/TestEnum.java

```diff
@@ -0,0 +1,9 @@
+package io.aexp.nodes.graphql.models;
```

Comment:
Could you please include the copyright information at the top of this file as well?

Reply:
Thanks, forgot this one.